tasks/tests/test_base.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from tasks.base import check_response
+from requests import Response
+
+
+def test_check_response_happy_path():
+    response = Response()
+    response.status_code = 200
+    response._content = '{"field": "value"}'
+
+    # Happy Path:
+    assert check_response(response) == ({"field": "value"}, True)
```

Comment:
I like to put docstrings under most functions if possible, so maybe a short sentence that describes what's being tested would be nice?

```python
def test_check_response_happy_path():
    """
    This tests that the check_response function will return the response content and a True value 
    when it receives a Response object with a 2xx status code and valid JSON in the content.
    """
```

This is also SUPER optional, but can be nice to have later on.